report/renderer/details-renderer.js

```diff
@@ -424,7 +424,21 @@ export class DetailsRenderer {
     if (!item.subItems) return fragment;
 
     const subItemsHeadings = headings.map(this._getDerivedsubItemsHeading);
-    if (!subItemsHeadings.some(Boolean)) return fragment;
+    if (!subItemsHeadings.some(Boolean) || !item.subItems.items.length) return fragment;
+
+    if (item.subItems.label) {
+      const rowEl = this._dom.createElement('tr');
+      const tdEl = this._dom.createChildOf(rowEl, 'td');
+      tdEl.append(this._renderText(item.subItems.label));
+      tdEl.append(this._dom.createElement('hr'));
+      rowEl.classList.add('lh-sub-item-row');
+      // Need a `td` per column so that background color of the table will
+      // span the width of the label row.
+      for (let i = 1; i < headings.length; i++) {
+        this._dom.createChildOf(rowEl, 'td');
+      }
```

Comment:
you're looking for [`colspan`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attr-colspan) :)

Reply:
hell yeah, thanks